Source/WebKit/UIProcess/ios/WKContentViewInteraction.h

```diff
@@ -445,14 +445,35 @@ struct ImageAnalysisContextMenuActionData {
     RetainPtr<WKSTextAnimationManager> _textAnimationManager;
 #endif
 
-    BOOL _isInSelectionInteraction;
+    enum class SelectionInteractionType : uint8_t {
+        None,
+        Touch,
+        ExtentPoint,
+        ExtentPointAndBoundary,
+    };
+
+    SelectionInteractionType _isInSelectionInteraction;
```

Comment:
Maybe this should be renamed to `_selectionInteractionType` for consistency?

Reply:
Good call